recipes/eelbrain/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "eelbrain" %}
+{% set version = "0.29.2" %}
+{% set sha256 = "23c72dfbb7ab458e6a24626a781ade91b541c0baa1e62fb3ee10cfdef711fdda" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  osx_is_app: True
+  skip: True  # [py!=36]
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - setuptools >=17
+  host:
+    - python
+    - python.app  # [osx]
+    - olefile  # [win or linux]
+    - pywin32-ctypes  # [win]
+    - pip
+    - cython >=0.21
+    - numpy 1.11.*
```

Is it necessary to force numpy 1.11 here?

```suggestion
    - numpy
```

---

Thanks @chrisburr, I was advised to build against the lowest compatible numpy version, hence `1.11`, but maybe that advice is outdated now? (see https://github.com/conda/conda-build/issues/2717#issuecomment-369316492)

---

I think this is now handled by default, [see here](https://conda-forge.org/docs/meta.html#building-against-numpy).

---

This says the default for Unix would currently be `1.9.3`, so I think it still needs to be pinned to `1.11`, right?

---

Yup, you're entirely correct. Sorry for the mix up.